### PR TITLE
cage now displays the background from the default desktop

### PR DESCRIPTION
**Pull request by DonatJR**
plus a few minor tweaks to fonts and labeler windows

example result: 
image

### CODE REVIEW

File: SharkCage/CageManager/CageDesktop.cpp [hunk 1]
```diff
@@ -54,6 +56,8 @@ bool CageDesktop::Init(HDESK &desktop_handle)
 		return false;
 	}
 
+	// FIXME close ctfmon.exe (input handler, leaks on win7 according to keepass)
```
Comment (SailReal) [inline SharkCage/CageManager/CageDesktop.cpp]: Maybe we should create an issue for that?

File: SharkCage/CageManager/CageLabeler.cpp [hunk 2]
```diff
@@ -74,6 +78,54 @@ bool CageLabeler::Init()
...
+		if (hwnd != hwnd_bg)
+		{
+			break;
+		}
+
+		PAINTSTRUCT ps;
+		HDC hdc = ::BeginPaint(hwnd, &ps);
+	
+		wchar_t wallpaper_path[MAX_PATH];
+		auto got_wp = SystemParametersInfo(SPI_GETDESKWALLPAPER, MAX_PATH, wallpaper_path, 0);
```
Comment (SailReal) [inline SharkCage/CageManager/CageLabeler.cpp]: Should be `::SystemParametersInfo...`?

File: SharkCage/CageManager/CageLabeler.cpp [hunk 3]
```diff
@@ -16,11 +16,15 @@ using namespace Gdiplus;
 #include <optional>
 #include <cstring>
 
-static bool DisplayTokenInCageWindow(HWND *hwnd);
+static bool DisplayTokenInCageWindow(HDC hdc);
 static bool GetBottomFromMonitor(int &monitor_bottom);
 static bool ShowExitButton(HWND &hwnd);
 static bool ShowConfigMetadata(HWND &hwnd);
 
+static HWND hwnd_lb;
```
Comment (SailReal) [inline SharkCage/CageManager/CageLabeler.cpp]: What means `hwnd_lb`? Can't figure out what the lb means 😅

Comment (DonatJR) [reply SharkCage/CageManager/CageLabeler.cpp]: the hwnd of the labeler (lb), do you prefer `hwnd_lbl` or `hwnd_labeler`?

**Comment by SailReal** [comment]
This branch could still be deleted or?

**Pull request merged**